examples/vue-wagmi/src/App.vue

```diff
@@ -42,13 +42,16 @@ const modal = useWeb3Modal()
 const state = useWeb3ModalState()
 const { setThemeMode, themeMode, themeVariables } = useWeb3ModalTheme()
 const events = useWeb3ModalEvents()
+
+console.log(themeMode)
```

Comment:
Any reason why this example was updated?

Reply:
I was testing examples and saw there were a lot of buttons in this example. So I removed a few.

Reply:
I popped them in there on purpose to see how each variant behaves, could we get them back please.